effectsize_example.Rmd

```diff
@@ -85,7 +100,7 @@ library(ggstance)   # for geom_pointrangeh()
 
 ```{r boilerplate, include = FALSE}
 set.seed(12)
-format_num <- function(...) format(..., digits = 2)
+format_num <- function(x, k = 2) format(round(x, k), nsmall = k)
```

Comment:
I think this works:

    nums = c(1, 12345, 1.2, 1.2345, 0.0012345)
    formatC(nums, 3, format = "fg", flag="#")

output:  `"1.00"    "12345."  "1.20"    "1.23"    "0.00123"`